### PR TITLE
hugepages: add some details on allocation

### PR DESCRIPTION
**Pull request by profnandaa**
This commit add a little more details on pre-allocation and dynamic allocation of huge pages on the node.

It also links to the Linux documentation for further details but gives a quick example on multiple page sizes that is not very obvious from the Linux documentation currently.

/cc. @derekwaynecarr

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
It's hard to find what you have changed. Is anything wrong when you are editing this page?

**Comment by tengqm** [comment]
This PR changed the EOL character?

File: content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 1]
```diff
@@ -1,116 +1,147 @@
...
+Kubernetes nodes must 
+[pre-allocate huge pages](https://www.kernel.org/doc/html/latest/admin-guide/mm/hugetlbpage.html)
+in order for the node to report its huge page capacity.
+
+A node can pre-allocate huge pages for multiple sizes, for instance,
+the following line in `/etc/default/grub` allocates `2*1GiB` of 1 GiB
+and `512*2 MiB` of 2 MiB pages:
+
+```
+GRUB_CMDLINE_LINUX="hugepagesz=1G hugepages=2 hugepagesz=2M hugepages=512"
```
Comment (derekwaynecarr) [inline content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md:27]: This was the primary technical change.  I do not know why the edit appears larger.

**Comment by profnandaa** [comment]
> It's hard to find what you have changed. Is anything wrong when you are editing this page?

Sorry, this was oversight on my end, failed to check my diff. This is caused by my Windows environment CRLF settings, fixing that rn.

**Comment by anjum-edge** [comment]
LGTM is having bugs due to this it is showing errors 
lookout for the new features and bugs

**Comment by sftim** [comment]
> LGTM is having bugs due to this it is showing errors lookout for the new features and bugs

I'm not sure I understand - is this nonsense?

File: content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 2]
```diff
@@ -14,15 +14,45 @@ by applications in a Pod. This page describes how users can consume huge pages.
...
+  cpu:                ...
+  ephemeral-storage:  ...
+  hugepages-1Gi:      2Gi
+  hugepages-2Mi:      1Gi
+  memory:             ...
+  pods:               ...
+```
 
+{{< note >}}
+For dynamically allocated pages (after boot), the Kubelet needs to be restarted
```
Comment (sftim) [inline content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md:53]:
```suggestion
For dynamically allocated pages (after boot), the kubelet needs to be restarted
```

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**